experiments/chat/chat/templates/chat/room_detail.html

```diff
@@ -13,6 +13,7 @@
     {% include "turbo/head.html" %}
 </head>
 <body>
+<a href="/">Home</a>
```

Comment:
Added a link to the index page here to trigger the issue around mounting and unmounting the stream element.